server/services/Mailer.js

```diff
@@ -3,24 +3,22 @@ const helper = sendgrid.mail;
 const keys = require('../config/keys');
 
 class Mailer extends helper.Mail {
-  constructor({ subject, recipients }, content) {
+  constructor(survey, content) {
     super();
 
     this.sgApi = sendgrid(keys.sendGridKey);
     this.from_email = new helper.Email('no-reply@emaily.com');
-    this.subject = subject;
+    this.subject = survey.subject;
     this.body = new helper.Content('text/html', content);
-    this.recipients = this.formatAddresses(recipients);
+    this.recipients = this.formatAddresses(survey.recipients);
```

Comment:
This is a case of overuse of parameter destructuring.  It's just confusing looking to have `{ subject, recipients }` as a function parameter. Like I don't know what that is, and it doesn't really save any work.

Instead, I just pass in  `survey`, which is what was being destructured, and then add `survey.` to the `subject` and `recipients`.

Works the same, but much easier to understand IMO.